src/compas_rhino/conversions/_geometry.py

```diff
@@ -179,3 +179,6 @@ def transform(self, T):
         else:
             M = T
         self.geometry.Transform(M)
+        if self.object:
+            self.object.CommitChanges()
+        compas_rhino.rs.Redraw()
```

Comment:
The exact behavior of `Redraw` in Rhino is not very clear to me, but isn't the redraw a bit inconvenient in terms of performance if you want to transform multiple objects? (ie. causing redraws on each, instead of being a one-off redraw at the end)

Reply:
yes, the redraw could be omitted indeed. will take it out...